ui/src/components/ResourceConfigForm/ParameterInput/ResourceDisplayNameInput.tsx

```diff
@@ -0,0 +1,51 @@
+import { FormHelperText, TextField } from "@mui/material";
+import { isFunction } from "lodash";
+import { ChangeEvent, memo } from "react";
+import { ParamInputProps } from "./ParameterInput";
+
+interface ResourceDisplayNameInputProps
+  extends Omit<ParamInputProps<string>, "definition"> {}
+
+const ResourceDisplayNameInputComponent: React.FC<
+  ResourceDisplayNameInputProps
+> = ({ value, readOnly, onValueChange }) => {
+  function handleChange(e: ChangeEvent<HTMLInputElement>) {
+    if (!isFunction(onValueChange)) {
+      return;
+    }
+
+    onValueChange(e.target.value);
+  }
+
+  return (
+    <TextField
+      defaultValue={value}
+      onChange={handleChange}
+      inputProps={{
+        "data-testid": "display-name-field",
+      }}
+      FormHelperTextProps={{
+        sx: { marginLeft: "-2px" },
+      }}
+      helperText={
+        <>
+          <FormHelperText component={"span"}>
+            Additional description for the resource.
+          </FormHelperText>
+          <br />
+        </>
+      }
```

Comment:
Whats going on here?  Seems to be just fine like this.
```suggestion
      helperText="Additional description for the resource."
```

Also I think it could just be "A friendly name for the resource"

Reply:
That's copy pasta from ResourceNameParamInput

Reply:
I think thats because that has error validation.